src/WebViewTypes.ts

```diff
@@ -432,6 +503,19 @@ export interface IOSWebViewProps extends WebViewSharedProps {
    * @platform ios
    */
   keyboardDisplayRequiresUserAction?: boolean;
+
+  /**
+	 * Specify a url scheme to intercept.
+	 */
+	urlScheme?: string;
+
+  /**
+   * Intercept a url scheme request. If you return a status, that means this is a response. If there
+   * is a method, that means that we should make the request from native code and pipe it. This is
+   * useful for requests such as images that won't encode well.
+   */
+  onUrlSchemeRequest?: (event: WebViewUrlSchemeRequest) => 
```

Comment:
Maybe share this prop with android? Same name / input and outputs

Reply:
See comment below about the differences between Android and iOS.